packages/perseus-editor/src/article-editor.tsx

```diff
@@ -116,10 +122,13 @@ export default class ArticleEditor extends React.Component<Props, State> {
         };
     }
 
-    _sections(): ReadonlyArray<RendererProps> {
-        return Array.isArray(this.props.json)
-            ? this.props.json
-            : [this.props.json];
+    _sections(): ReadonlyArray<PerseusRenderer> {
+        const sections = Array.isArray(this.props.json)
+            ? this.props.json.map((section) => {
+                  return convertDeprecatedWidgets(section as PerseusRenderer);
+              })
+            : [convertDeprecatedWidgets(this.props.json as PerseusRenderer)];
+        return sections;
```

Comment:
I'm not a big fan of this having to run so frequently. I wonder if it would be better to be converting the json in the main repo by exporting the util from the editor package.